projects/sound-open-firmware/Dockerfile

```diff
@@ -20,39 +20,57 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 
-RUN apt-get -y update && \
-	    apt-get install -y \
-	    	autoconf \
-		bison \
-		build-essential \
-		flex \
-		gawk \
-		gettext \
-		git \
-		gperf \
-		help2man \
-		libncurses5-dev \
-		libssl-dev \
-		libtool \
-		libtool-bin \
-		pkg-config \
-		software-properties-common \
-		texinfo \
-		udev \
-		cmake \
-		libglib2.0-dev
-
-ARG CLONE_DEFAULTS="--depth 5"
-
-# Use ToT alsa utils for the latest topology patches.
-RUN cd /tmp && \
-git clone $CLONE_DEFAULTS https://github.com/thesofproject/alsa-lib.git && \
-git clone $CLONE_DEFAULTS https://github.com/thesofproject/alsa-utils.git && \
-cd /tmp/alsa-lib && ./gitcompile &&  make install && \
-cd /tmp/alsa-utils && ./gitcompile &&  make install
-
-ARG GITHUB_SOF=https://github.com/thesofproject
-
-RUN cd $SRC && git clone $CLONE_DEFAULTS $GITHUB_SOF/sof
-WORKDIR sof
+# Base packages
+RUN apt-get -y update
+RUN apt-get install -y \
+      gettext git libc6-dev-i386 libglib2.0-dev libncurses5-dev \
+      libtool ninja-build python3-pip
+RUN pip3 install west
+
+# Zephyr SDK:
+#
+# Zephyr doesn't provide a clean "get the latest version" URL, but
+# note that the use of the /latest/ path component at least ensures
+# this will fail on a new release.
+ARG SDK_VER=0.16.1
```

Comment:
Can this be overriden on some --arg command line? If yes maybe say it for the Docker noobs among us.

Reply:
At the docker level, yes.  `docker build --build-arg SDK_VER=0.16.2` would have the expected effect.  I have no idea if that's wired meaingfully to the oss-fuzz infrastructure or not.

Really this is just here to define a local variable for clarity.   Docker has two variable tyes: ARG works like this and can be set externally, but only works within the dockerfile itself.  ENV cannot be set externally, but also persists into the default environment variables of processes spawned in the container, which is undesirable here.

Reply:
Bleh if the versions weren't part of the release file name this would be much easier. Seems like a [common](https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8) problem.

Reply:
I think we will have to leave this as it, not much hope for fixing it without some additional logic